scala-generator/src/main/scala/models/play/components/Controllers.scala

```diff
@@ -0,0 +1,69 @@
+package scala.models.play.components
+
+import cats.data.{Validated, ValidatedNel}
+import cats.implicits._
+import io.apibuilder.spec.v0.models._
+import scala.generator._
+import scala.models.play.Helpers._
+
+class Controllers(service: ScalaService) extends Component {
```

Comment:
100% generic with `F[_]`. It would allow this type to be reused for other frameworks. To be useful, it requires a mapping from `HTTP${code}` to the proper result type. Might move that out of this object to allow play mapping, http4s mapping, ...